eric_sse/entities.py

```
@@ -1,55 +1,64 @@
+    def __init__(self, queues_repository: AbstractMessageQueueRepository):
+        self.__listeners: dict[str: MessageQueueListener] = {}
+        self.__queues: dict[str: Queue] = {}
+        self.__queues_factory = queues_repository
 
-    async def start(self) -> None:
-        self.start_sync()
 
-    def start_sync(self) -> None:
-        logger.debug(f"Starting listener {self.id}")
-        self.__is_running = True
+    def add_listener(self) -> MessageQueueListener:
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)